### DISCUSSION TITLE
Search in a select

### DISCUSSION THREAD

**Discussion by ghost** [taiga-family/taiga-ui#1176] [Q&A] [answered]
According to the documentation, the  [select](https://taiga-ui.dev/components/select/API?tuiMode=null) component is missing a search. How can search in select similarly to MultiSelect?

**Comment by waterplea** [comment, marked as answer]
You're looking for [ComboBox](https://taiga-ui.dev/components/combo-box).

**Comment by ghost** [comment]
@waterplea Unfortunately the DataListWrapper example does not work on the stackblitz

**Reply by waterplea** [reply]
> @waterplea Unfortunately the DataListWrapper example does not work on the stackblitz

That's because Ivy for Taiga UI doesn't work on StackBlitz. Try to just copy it to your project. The example uses pipes for filtering that only work with Ivy and ngcc on StackBlitz fails on huge projects.
